src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +50,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(e -> e.getSex() == Person.Sex.MAN
+                        && e.getAge() >= fromAge
+                        && e.getAge() <= toAge)
```

Comment:
lets name params more informative
```suggestion
                .filter(person -> person.getSex() == Person.Sex.MAN
                        && person.getAge() >= fromAge
                        && person.getAge() <= toAge)
```
check other methods as well